### PR TITLE
feat(ccl): create an issue comment for notifying user

### LINKED ISSUE #3733: Conventional Commit Linter: Always lint PR Titles
**Is your feature request related to a problem? Please describe.**

The current linter implementation scans commit messages for single-commit branches, and PR titles for multi-commit branches.

If someone makes a quick change without following the conventional commit standard, they cannot easily fix it via the GitHub UI. Instead, they need to go back to their development environment and amend the commit or add a spurious commit to sidestep the linter.

This friction especially impacts new contributors unfamiliar with conventional commits, folks trying to help with a quick change via the GitHub UI, or developers still building muscle memory on using conventional commit forms.

 **Describe the solution you'd like**

Change the default behavior or add a configuration option to always and only scan the PR title.

This makes the contribution experience easier, at the cost that the maintainer will need to be more thoughtful about checking the commit message before merging.

 **Describe alternatives you've considered**

Disable conventional commit linting.

### PR DESCRIPTION
**Pull request by tmatsuo**
about mismatched commit messages

fixes #3733

### CODE REVIEW

File: packages/conventional-commit-lint/src/utils.ts [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+      logger.info(`PR title: ${pull_request.title}`);
+      const owner = pull_request.base.repo.owner?.login;
+      const repo = context.payload.repository.name;
+      const prNumber = pull_request.number;
+      const installationId = context.payload.installation?.id;
+      const message =
+        '🤖 I detect that the PR title and the commit message' +
+        " differ and there's only one commit. To use the PR title for the" +
+        " commit history, you can use Github's automerge feature with" +
+        ' squashing, or use `automerge` label. Good luck human!\n\n' +
```
Comment (bcoe) [inline packages/conventional-commit-lint/src/utils.ts:152]: 😆 I like the "Good luck human!".

File: packages/conventional-commit-lint/test/conventional-commit-lint.ts [hunk 2]
```diff
@@ -16,58 +16,99 @@
...
         snapshot(body);
         return true;
       })
       .reply(200);
+    await probot.receive({name: 'pull_request', payload, id: 'abc123'});
+    requests.done();
+    sinon.assert.notCalled(addOrUpdateIssueCommentStub);
+  });
 
+  it('adds a comment when the commit message and the PR title defer', async () => {
```
Comment (bcoe) [inline packages/conventional-commit-lint/test/conventional-commit-lint.ts:86]: nit: should be "differ".

File: packages/conventional-commit-lint/src/utils.ts [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+  await context.octokit.checks.create(checkParams);
+}
+
+// export function lint_wrapper(message: string, rules: any): ILint {
+//   // support "foo!: message" syntax, see:
+//   // https://github.com/conventional-changelog/commitlint/pull/767
+//   message = message.replace('!:', ':');
+
+//   return lint(message.toLowerCase(), rules);
+// }
```
Comment (bcoe) [inline packages/conventional-commit-lint/src/utils.ts:213]: Let's delete this logic if it's not used.

**Review by bcoe — CHANGES_REQUESTED** [review]
Left a few nits, this looks great in general.

File: packages/conventional-commit-lint/src/utils.ts (hunk 3 above)
Comment (tmatsuo) [reply packages/conventional-commit-lint/src/utils.ts]: Done

**Review by bcoe — APPROVED** [review]

**Pull request merged**